controllers/tasksController.js

```diff
@@ -0,0 +1,47 @@
+const taskQuery = require('../models/tasks')
+/**
+ * Creates new task
+ *
+ * @param req {Object} - Express request object
+ * @param req.body {Object} - Task object
+ * @param res {Object} - Express response object
+ */
+const addNewTask = async (req, res) => {
```

Comment:
Isn't there any validation added for the request data?
Any user can thus add any long request body and it will just land in the DB!

Reply:
Also, can you add tests for those validators and other functions defined in PR?

Reply:
I will add the validations, regarding tests I need to check .. let me get these validations in place and then lets check on this tests